### PR TITLE
Add error message for improperly configured server.listen

### PR DESCRIPTION
**Pull request by sjrct**
## Checklist
- [x] All new and existindocs/docs/technical-docs/testing.rstg **tests are passing**
- [x] I have written **proper commit message(s)**
      

## What changes does this Pull Request introduce?
Add error message for improperly configured `server.listen`

## Why is this necessary?
Otherwise the error message is quite cryptic, what I saw was ultimately:
```
AttributeError: 'int' object has no attribute 'strip'
```

### CODE REVIEW

File: isso/__init__.py [hunk 1]
```diff
@@ -295,11 +295,14 @@ def main():
...
+    elif conf.get("server", "listen").startswith("unix://"):
         sock = conf.get("server", "listen").partition("unix://")[2]
         try:
             os.unlink(sock)
         except OSError as ex:
             if ex.errno != errno.ENOENT:
                 raise
         wsgi.SocketHTTPServer(sock, make_app(conf)).serve_forever()
+    else:
+        logger.error("server.listen must specif a protocol of http:// or unix://")
```
Comment (BBaoVanC) [inline isso/__init__.py:307]: nit: should say "specify" instead of "specif"

**Review by BBaoVanC — APPROVED** [review]
Looks good, I think there's a small typo though

File: isso/__init__.py (hunk 1 above)
Comment (sjrct) [reply isso/__init__.py]: Fixed

**Comment by ix5** [comment]
@BBaoVanC if you're satisfied, I would like to merge this.

**Pull request merged**